packages/core/src/render3/node_manipulation.ts

```diff
@@ -844,23 +844,23 @@ function executeActionOnContainer(
 
 
 /**
- * `executeActionOnElementContainer` performs an operation on the ng-container node and its child
- * nodes as specified by the `action` (insert, detach, destroy).
+ * `executeActionOnElementOrIcuContainer` performs an operation on the ng-container node and its
+ * child nodes as specified by the `action` (insert, detach, destroy).
  *
  * @param renderer Renderer to use
  * @param action action to perform (insert, detach, destroy)
  * @param lView The LView which needs to be inserted, detached, destroyed.
- * @param tElementContainerNode The TNode associated with the ElementContainer.
+ * @param tContainerNode The TNode associated with the `ElementContainer` or `IcuContainer`.
  * @param renderParent parent DOM element for insertion/removal.
  * @param beforeNode Before which node the insertions should happen.
  */
-function executeActionOnElementContainer(
+function executeActionOnElementOrIcuContainer(
```

Comment:
Nit: the name sounds like elements or icu containers rather than element containers or icu containers

```suggestion
function executeActionOnElementContainerOrIcuContainer(
```